static/js/formManager.js

```diff
@@ -1,7 +1,7 @@
 const FormManager = (function() {
```

Comment:
this should be in camelCase starting with lower case

Reply:
Resolved!